README.md

```diff
@@ -237,7 +237,11 @@ prettier.format(source, {
   jsxBracketSameLine: false,
 
   // Which parser to use. Valid options are 'flow' and 'babylon'
-  parser: 'babylon'
+  parser: 'babylon',
+
+  // Whether to add a semicolon at the end of every line (semi: true),
+  // or only at the beginning of lines that may introduce ASI failures (semi: false)
+  semi: true
```

Comment:
Ok, yeah the CLI arg is the main thing I was thinking about. I guess it's fine if the API option is different and the CLI one is `--no-semi`.